next-version/README.md

```diff
@@ -12,11 +12,13 @@ Example generating the next version from a repo with version tags with last rele
     tag-prefix: my-project-
 ```
 
-If for example there are breaking changes, the output will be `v2.0.0`
+If for example there are breaking changes, the output will be `v2.0.0`, `v2.0`, and `v2`.
 
 ## Outputs
 
-`next-version`: Returns Semver next version, with leading v
+`next-version`: Returns Semver next version, with leading v. E.g.: `v3.4.1`
+`next-version-major`: Returns only the major version, with leading v. E.g.: `v3`
+`next-version-major-minor`: Returns major and minor version, with leading v. E.g.: `v3.4`
```

Comment:
I think the docs and the implementation doesn't match, we have  a `next-version-` prefix here in the docs and 
 only `next-` in the implementation, sorry!

https://github.com/newrelic/release-toolkit/blob/af3c65d22a339b53b9579b7c0fa998170e3d78fd/src/app/nextversion/nextversion.go#L34-L35

Reply:
I should be solved on [#216dc52](https://github.com/newrelic/release-toolkit/pull/188/commits/216dc52929dba8f34fda73ca2df5b8fc42abf6c8)